WordPress/src/main/java/org/wordpress/android/util/QuickStartUtils.kt

```diff
@@ -176,15 +185,43 @@ class QuickStartUtils {
         ) {
             val siteId = site.id.toLong()
 
-            if (quickStartStore.getQuickStartCompleted(siteId) || isEveryQuickStartTaskDone(quickStartStore)) {
+            if (quickStartStore.getQuickStartCompleted(siteId) || isEveryQuickStartTaskDone(quickStartStore) ||
+                    quickStartStore.hasDoneTask(siteId, task)) {
                 return
             }
 
             quickStartStore.setDoneTask(siteId, task, true)
+            AnalyticsTracker.track(getTaskCompletedTracker(task))
 
             if (isEveryQuickStartTaskDone(quickStartStore)) {
+                AnalyticsTracker.track(AnalyticsTracker.Stat.QUICK_START_COMPLETE_ALL_TASKS)
```

Comment:
We can remove the unnecessary `AnalyticsTracker` qualification from `AnalyticsTracker.Stat.QUICK_START_COMPLETE_ALL_TASKS ` to produce `AnalyticsTracker.track(AnalyticsTracker.Stat.QUICK_START_COMPLETE_ALL_TASKS)` for simplicity.